src/msg/async/Event.cc

```diff
@@ -48,6 +52,10 @@ class C_handle_notify : public EventCallback {
  public:
   C_handle_notify(EventCenter *c, CephContext *cc): center(c), cct(cc) {}
   void do_request(uint64_t fd_or_id) override {
+#ifdef __linux__
+    eventfd_t value;
+    read(fd_or_id, &value, sizeof(value));
```

Comment:
instead of `read()`, i'd suggest use `eventfd_read()` instead, simpler this way. also, we might want to check the return value of `read()` or `eventfd_read()`.

Reply:
eventfd_read() is a glibc-specific wrapper for read(), but I don't know why it exists, a raw read() is simple enough. But being a glibc extension means you have to check for its availability at configure time and have a fallback to read(). I don't see the point in that.

Reply:
@tchaikov, do you agree or do you insist on using this non-standard glibc function?

Reply:
@MaxKellermann sorry for the latency. i don't insist. 

> also, we might want to check the return value of read() or eventfd_read().

did you get a chance to address this comment?

Reply:
Also note that my patch ignores the value of the `eventfd_t`. The whole point of the eventfd is getting epoll wakeups; the actual value is not interesting; the do the read only to reset its readiness.